.github/workflows/continous-integration-docker.yml

```
@@ -581,7 +581,7 @@ jobs:
     name: lightwalletd tip send
     needs: lightwalletd-full-sync
     uses: ./.github/workflows/deploy-gcp-tests.yml
-    if: ${{ !cancelled() && !failure() && ((github.event_name == 'push' && github.ref_name == 'main') || github.event.inputs.run-lwd-send-tx == 'true') }}
+    if: ${{ !cancelled() && !failure() && github.event.inputs.regenerate-disks != 'true' && github.event.inputs.run-full-sync != 'true' && github.event.inputs.run-lwd-sync != 'true' || github.event.inputs.run-lwd-send-tx == 'true' }}
```

We should update this too if we remove the manual workflow dispatch:

```suggestion
    if: ${{ !cancelled() && !failure() && github.event.inputs.regenerate-disks != 'true' && github.event.inputs.run-full-sync != 'true' && github.event.inputs.run-lwd-sync != 'true' }}
```

---

Good catch!

---

Fixed by: [fix(actions): remove references to the workflow_dispatch](https://github.com/ZcashFoundation/zebra/pull/6040/commits/fbba1d159393a0e45b3bda199a8ff3b76ff07e46)